### PR TITLE
Add install commands and __init__.py

### PR DESCRIPTION
**Pull request by mehditlili**
After installing my package, I noticed that an __init__.py was missing and python was unable to import correctly.

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -59,4 +59,14 @@ set_target_properties(mycpplib PROPERTIES
...
   ${Boost_LIBRARIES}
+
+)
+
+install(TARGETS mycpplib
+  ARCHIVE DESTINATION ${CATKIN_PACKAGE_PYTHON_DESTINATION}
+  LIBRARY DESTINATION ${CATKIN_PACKAGE_PYTHON_DESTINATION}
 )
+
+file(GLOB py_files "scripts/*.py")
```
Comment (luator) [inline CMakeLists.txt]: To keep it minimalist, I would prefer to not use the `file` command but just use the explicit filename.

File: CMakeLists.txt [hunk 2]
```diff
@@ -59,4 +59,14 @@ set_target_properties(mycpplib PROPERTIES
...
+install(PROGRAMS ${py_files}
```
Comment (luator) [inline CMakeLists.txt]: Please use the `catkin_install_python` macro to install python scripts. See http://docs.ros.org/kinetic/api/catkin/html/howto/format2/installing_python.html

**Review by luator — CHANGES_REQUESTED** [review]
Thanks for your input, it is very appreciated!

I have some comments on the installation of the python script (see below).
And a question to the `__init__.py` file: Is this really necessary for some reason? If not, I would prefer to remove it to keep this example as minimal as possible (i.e. only including parts that are necessary for getting the Boost Python bindings to work).

**Comment by mehditlili** [comment]
When I tried without the __init__.py, the mycpplib.so is installed but can't be imported in Python.

**Comment by luator** [comment]
Sorry for the late response, looks good now :). Thanks again.

**Pull request merged**